src/server/modes/opensource/app.ts

```diff
@@ -71,6 +90,89 @@ function initDataLensApp({
     afterAuth.push(xDlContext(), getCtxMiddleware());
 }
 
+function initZitadel({beforeAuth}: {beforeAuth: AppMiddleware[]}) {
```

Comment:
I think it will be better to extract this code from `app.ts` because not all the customers need it.

Reply:
Aggree, extracted, the same approach was used with charts as I see. The code became cleaner